### DISCUSSION TITLE
issue running GENIE3 on calculation cluster and scenic on computer

### DISCUSSION THREAD

**Discussion by vbouther** [aertslab/SCENIC#263] [R]
Hello scenic community, 
I am trying to run scenic on a 900*4500 matrix. I had to run genie3 on a cluster as my computer was not powerful enough but I was hoping to run the rest of the code on my computer, as it is very challenging to install packages on the cluster. 
I've downloaded the weight matrix generated by GENIE3 on the cluster as a RData file and reload it on my rstudio session, it worked fine. However, when I attempt to run the very first step of SCENIC, here is the error message: 
runSCENIC_1_coexNetwork2modules(scenic_opts)
Error in .local(object, ...) : 
  File 'int/1.4_GENIE3_linkList.Rds' does not exist.

I have reloaded the scenic_opts file but didnt work any better. In my int folder I only have: 
ls int
1.1_genesKept.Rds	1.2_corrMat.Rds		scenic_opts.Rds

I feel like there are some files that have been created while running GENIE3 on the cluster that are needed to run scenic and I don't know which one and how to get them (I cannot find any int folder on the server).

Has anyone encountered this issue before?
Thank you very much!!

**Comment by EAC-T** [comment]
I'm having the same problem, I didn't use a cluster, I only used my laptop.
Have you find out what the issue was? I hope you can help. 
Thank you

**Comment by apal6** [comment]
Hi,

I am having the same issue, were you able to fix it?

Thank you so much!
